x-pack/plugins/cloud_security_posture/public/pages/rules/rules_table_header.tsx

```diff
@@ -283,11 +283,11 @@ const CurrentPageOfTotal = ({
     const bulkSelectedRules = selectedRules.map((e: CspBenchmarkRulesWithStatus) => ({
       benchmark_id: e?.metadata?.benchmark.id,
       benchmark_version: e?.metadata.benchmark.version,
-      rule_number: e?.metadata.benchmark.rule_number,
+      rule_number: e?.metadata.benchmark.rule_number || 'Unknown',
```

Comment:
we need to make sure this is a valid value from the API perspective. the API might throw errors on those cases and we don't want to crash the page. the better solution if we dont have a rule number would be to just not call the api.

@CohenIdo

Reply:
The source of the concern of missing `rule_number` is that we introduce this field in 8.7. 
Since packages are automatically upgraded we can be sure that  `metadata.benchmark.rule_number` will exist. 

[Reference](https://github.com/elastic/kibana/blob/c05b8935d93e4ef656ee6f1e1d1db714fdc9d609/x-pack/plugins/cloud_security_posture/common/types/rules/v3.ts#L23).